hw_4(decor).py

```diff
@@ -0,0 +1,57 @@
+# 1
+def leave (func):
+    def wrapper(ar):
+        result=func(ar)
+        b=100%ar
```

Comment:
we should check the result of function, not argument, so change it to `b = 100 % result`